Taskfile.yml

```diff
@@ -44,6 +46,13 @@ tasks:
         <(cd '{{.OUTPUT_DIR}}' && tar cf - {{.CHECKSUM_TAR_BASE_ARGS}} . | md5sum)
         "{{.CHECKSUM_FILE}}"
 
+  serve:
+    deps:
+      - "http-server"
+      - "docs"
```

Comment:
If I'm not mistaken, task runs dependencies in parallel, so the order here doesn't affect execution. So I'd say we can alphabetize them.